### PR TITLE
Issue 16 - added linting

### PR DESCRIPTION
**Pull request by avrahamgoldman**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -13,7 +18,13 @@
     "url": "https://github.com/SparkPost/node-sparkpost-cli/issues"
   },
   "homepage": "https://github.com/SparkPost/node-sparkpost-cli",
-  "engines" : { "node" : ">=4.0.0" },
+  "engines": {
+    "node": ">=4.0.0"
```
Comment (rajumsys) [inline package.json]: just fyi, i've verified with 4.0.0 and found it's working (with the #31)

File: package.json [hunk 2]
```diff
@@ -24,10 +35,25 @@
     "sparkpost": "^2.1.2",
     "yargs": "^6.6.0"
   },
+  "devDependencies": {
+    "eslint": "^3.15.0",
```
Comment (rajumsys) [inline package.json]: linting rules may in major versions. as we use default, i think we should change it to `~`.

**Comment by rajumsys** [comment]
@avrahamgoldman 
- other than the small comment, your changes look good
- i've resolve the conflicts (due to #31) and pushed another minor change. 

once you review these two, it's ready to  🚢

File: package.json (hunk 2 above)
Comment (avrahamgoldman) [inline package.json]: Makes sense. I'll make that change

**Pull request merged**